.travis.yml

```diff
@@ -39,6 +46,7 @@ script:
 - bundle exec make test
 - git diff --exit-code
 - git diff --cached --exit-code
+- sudo lsof | grep dpkg || true
```

Comment:
What's this for?

Reply:
I was getting inconsistent issues with *something* locking the dpkg lock file while attempting to install Chef DK stuff for the serverspecs tests.  Once I added this line to see what it could have been, I couldn't reproduce the error.  I would like to keep it around for a bit until we are no longer seeing issues with *something* locking the dpkg lock file.